code/modules/organs/organ.dm

```diff
@@ -328,6 +328,9 @@
 			admin_attack_log(user, owner, "Removed a vital organ ([src])", "Had a a vital organ ([src]) removed.", "removed a vital organ ([src]) from")
 		owner.death()
 
+	if(item_upgrades.len > 0)
+		owner.mutation_index -= 1
```

Comment:
`Length()` is noticeably faster than `.len`, or so I've read somewhere. It's also usually used via `LAZYLEN()` define.
```suggestion
	if(LAZYLEN(item_upgrades))
		owner.mutation_index--
```

Reply:
Interesting. I need to start using those defines more.